### PR TITLE
Update jsonpatch.md

### LINKED ISSUE #25286: Missing code in JsonPatch in ASP.NET Core web API
The "Action Method Code" has two code listings that are empty, just after the paragraph: "This code from the sample app works with the following Customer model:" just about in the middle of the page.. It doesn't look right :-)

image

Also, I got to that page after a Google search and I'm unsure if I can use this page's content when I'm building my Web API Core for use with my custom JavaScript SPA website - or if it's for when building an web app from the Visual Studio template, that runs a Web API as backend. Maybe the page should have a "this is for when you have xx and are wanting to do yy" section at the top?.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 593f528c-61b9-7021-335d-c1c0ec1814d1
* Version Independent ID: 6a88d237-1a03-b865-e3c5-20ceedef2a1f
* Content: [JsonPatch in ASP.NET Core web API](https://docs.microsoft.com/en-us/aspnet/core/web-api/jsonpatch?view=aspnetcore-5.0#feedback)
* Content Source: [aspnetcore/web-api/jsonpatch.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/web-api/jsonpatch.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-webapi**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #25286

### CODE REVIEW

File: aspnetcore/web-api/jsonpatch.md [hunk 1]
```diff
@@ -134,9 +134,9 @@ Here's an example:
 
 This code from the sample app works with the following `Customer` model:
 
-[!code-csharp[](jsonpatch/samples/6.x/api/Models/Customer.cs)]
+[!code-csharp[](jsonpatch/samples/3.x/api/Models/Customer.cs)]
```
Comment (Rick-Anderson) [inline aspnetcore/web-api/jsonpatch.md:137]:
```suggestion
[!code-csharp[](jsonpatch/samples/6.x/api/Models/Customer.cs)]
```

File: aspnetcore/web-api/jsonpatch.md [hunk 2]
```diff
@@ -134,9 +134,9 @@ Here's an example:
...
 
-[!code-csharp[](jsonpatch/samples/6.x/api/Models/Order.cs)]
+[!code-csharp[](jsonpatch/samples/3.x/api/Models/Order.cs)]
```
Comment (Rick-Anderson) [inline aspnetcore/web-api/jsonpatch.md:139]:
```suggestion
[!code-csharp[](jsonpatch/samples/6.x/api/Models/Order.cs)]
```

**Comment by Rick-Anderson** [comment]
@serpent5  it's the red dot problem. @guardrex  what's that called?

image

**Comment by guardrex** [comment]
BOM ... Byte Order Mark ... common and harmless AFAIK ... you can keep or delete them without harm AFAIK.

**Comment by Rick-Anderson** [comment]
I wonder why the code was not showing up. It showed up in the build.
This is current production
image

**Pull request merged**